### PR TITLE
Add customer getter

### PR DESCRIPTION
**Pull request by hzxcaryn**
## Changes
- Added customer getter
- Added `datastore.ts`, a file that contains datastore wrappers. `getWithId` strips the KEY_SYMBOL from the GetResponse object, and adds an `id` field to allow better typing throughout the entire repo.
- Changed some exports to default exports (for the files that I touched in this PR, will change the rest in other PRs)

## How to test
Fetch my branch
```
cd /server
yarn
yarn start:dev
```
Send a request to `http://localhost:5000/customers/5634161670881280` (our dear sample customer)

### CODE REVIEW

File: server/src/storage/datastore.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Datastore} from '@google-cloud/datastore';
+
+const datastore = new Datastore();
+
+const getWithId = async (kind: string, id: string) => {
```
Comment (ifhuang) [inline server/src/storage/datastore.ts:21]: can you add comment for this?

File: server/src/storage/customers.ts [hunk 2]
```diff
@@ -14,9 +14,12 @@
...
 
-import {Datastore} from '@google-cloud/datastore';
-import {Guid} from 'guid-typescript';
+import {CUSTOMER_KIND} from '../constants/kinds';
+import {CustomerResponse} from '../interfaces';
+import {getWithId} from './datastore';
 
-const datastore = new Datastore();
+const getCustomer = async (customerId: string): Promise<CustomerResponse> => {
+  return getWithId(CUSTOMER_KIND, customerId);
```
Comment (ifhuang) [inline server/src/storage/customers.ts:22]: You already have =>, do you still need {return;} block?

File: server/src/storage/datastore.ts (hunk 1 above)
Comment (ifhuang) [inline server/src/storage/datastore.ts:21]: can => be removed?

Comment (hzxcaryn) [reply server/src/storage/datastore.ts:21]: What do you mean by this? Use normal functions instead?

File: server/src/storage/customers.ts (hunk 2 above)
Comment (hzxcaryn) [reply server/src/storage/customers.ts:22]: Oh yea I will remove that!

File: server/src/storage/datastore.ts (hunk 1 above)
Comment (hzxcaryn) [reply server/src/storage/datastore.ts:21]: ok!

File: server/src/interfaces.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ */
+export interface CustomerPayload {
+  gpayId: string;
+  contactNumber: string;
+  address: string;
+}
+
+/**
+ * CustomerResponse Interface that contains the fields of the Response that
+ * client side would recieve.
```
Comment (karenfrilya97) [inline server/src/interfaces.ts:29]: nit: typo in 'receive' hahah

Comment (hzxcaryn) [reply server/src/interfaces.ts:29]: Wah my spelling cmi HAHHA nice catch!

File: server/src/storage/datastore.ts (hunk 1 above)
Comment (hzxcaryn) [reply server/src/storage/datastore.ts:26]: Resolving, had a chat on Chat!

**Pull request merged**